lib/github_service/commands/remove_label.rb

```diff
@@ -1,19 +1,31 @@
 module GithubService
   module Commands
     class RemoveLabel < Base
+      UNREMOVABLE = %w[jansa/yes jansa/no]
```

Comment:
This one could probably be a a little more programmatic, but left it like this for now.

Side note: probably removing the ability to remove `*/backported` would probably be a good idea in general.

Reply:
> Side note: probably removing the ability to remove `*/backported` would probably be a good idea in general.

:+1: Sounds good to me.  What do you think @simaishi ?

Reply:
Agreed! The only time `/backported` label should get removed is when we revert the backport, in which case I manually set appropriate labels.

Reply:
We should probably disable adding the label too, right?  Anyone that can do the backporting should be able to manually add the label (not through bot commands)

Reply:
Cool, I will follow some of this up in another PR.

👍